### PR TITLE
create modal

### PR DESCRIPTION
**Pull request by Ntare22**
* Create js function to dynamically add the projects
* Create modal for the project
* Add css for the modal

### CODE REVIEW

**Review by TedLivist — CHANGES_REQUESTED** [review]
## STATUS: INVALID CODE REVIEW REQUEST 🛑 🛑

Hi @Ntare22,

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a Pull Request with no descriptive summary.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: index.js [hunk 1]
```diff
@@ -25,3 +25,174 @@ for (let i = 0; i < colapseBtn.length; i++) {
...
+    viewSource: 'https://github.com/Ntare22/personal-portfolio-2.0'
+  },
+]
+
+function modalExit() {
+  document.getElementById('modal-container').style.display = 'none';
+}
+
+function viewProject(project) {
+  // console.log('clicked on view project', project);
```
Comment (carlos-ssh) [inline index.js]: - Keep professionalism and clean code in your projects. Remove commented code.

File: index.js [hunk 2]
```diff
@@ -25,3 +25,174 @@ for (let i = 0; i < colapseBtn.length; i++) {
...
+  project.projectTags.map(tag => {
+    let newListTag = document.createElement('li');
+    newListTag.classList = 'project-tag';
+    newListTag.innerHTML = tag;
+    parentTag.appendChild(newListTag)
+  });
+  
+
+  // const title = document.getElementById('modal-title');
+  // const img = document.getElementById('modal-img')
```
Comment (carlos-ssh) [inline index.js]: - Keep professionalism and clean code in your projects. Remove commented code.

File: index.js [hunk 3]
```diff
@@ -25,3 +25,174 @@ for (let i = 0; i < colapseBtn.length; i++) {
...
+    projectImg.setAttribute('src', project.img);
+    projectImg.setAttribute('alt', project.imgAlt);
+    
+    projectContainer.appendChild(projectImg);
+    
+    const projectInfo = document.createElement('div');
+    projectInfo.classList = 'project-info';
+    
+    //// Project Info \\\
+    // project title
```
Comment (carlos-ssh) [inline index.js]: [OPTIONAL]
- Comments are helpful in a development environment try to keep your code clean once that you request a code review... Your code explains by itself what it do 😁.

File: index.js [hunk 4]
```diff
@@ -25,3 +25,174 @@ for (let i = 0; i < colapseBtn.length; i++) {
...
+    const projectTitle = document.createElement('h2');
+    projectTitle.innerHTML = project.projectTitle;
+    projectTitle.classList = 'project-title';
+    projectInfo.appendChild(projectTitle);
+    projectContainer.appendChild(projectInfo);
+
```
Comment (carlos-ssh) [inline index.js:106]: - Your ``README`` file is missing information.
Screen Shot 2021-10-06 at 12 26 58

**Review by carlos-ssh — CHANGES_REQUESTED** [review]
Hi @Ntare22 , 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by safafa — APPROVED** [review]
## STATUS: APPROVED 🟢 👏
Hello, @Ntare22  great job working on the required changes 👍 💯 

- [ ] [OPTIONAL] You're missing the Javascript linter. Since the first reviewer didn't  mention it I marked it as optional but I highly recommend you to add it check [this link ](https://github.com/microverseinc/linters-config/tree/master/html-css-js)

Now you can merge this branch 💪 😄
HAPPY CODING!! AND GOOD LUCK WITH THE NEXT ONES 🤞 🍀

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**